public/resources/index.ejs

```diff
@@ -5,10 +5,10 @@
         <div>
             <h1 class="capitalize">{{category}}</h1>
```

Comment:
We will keep the `h1` for now, unless anyone has any objections, this is the `Education `, `Development `, `Community ` headers for each column.